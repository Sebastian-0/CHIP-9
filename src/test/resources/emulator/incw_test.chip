
ldx BC, 255
ldx DE, 65535
ldx HL, 10

inx BC
inx DE
inx HL
inx HL

sout BC
sout DE
sout HL

hcf