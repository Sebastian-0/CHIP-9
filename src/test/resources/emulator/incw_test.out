256
0
12